audits/performance/first-meaningful-paint.js

```diff
@@ -28,25 +28,32 @@ class FirstMeaningfulPaint {
     return 'Has a good first meaningful paint';
   }
 
+  /**
+   * Audits the page to give a score for First Meaningful Paint.
+   * @see  https://github.com/GoogleChrome/lighthouse/issues/26
+   * @param  {Object} inputs The inputs from the gather phase.
```

Comment:
Actually you make a good point; we need to decide our general approach to docs, i.e. are we using jsdoc, esdoc, any kind of doc?